great_expectations/core/usage_statistics/schemas.py

```diff
@@ -15,7 +15,7 @@
 )
 
 SCHEMA: str = "https://json-schema.org/draft/2020-12/schema"
-ALLOWED_VERSIONS: List[str] = ["1.0.0", "1.0.1"]
+ALLOWED_VERSIONS: List[str] = ["1.0.0", "1.0.1", "1.0.2"]
```

Comment:
Per discussion with @tannerbeam, this doesn't really matter but I'm going to just increment as I go